pyaedt/generic/spisim.py

```diff
@@ -0,0 +1,208 @@
+        if is_linux:  # pragma: no cover
+            command.append("&")
+            with open(out_processing, "w") as outfile:
+                subprocess.Popen(command, env=my_env, stdout=outfile, stderr=outfile).wait()  # nosec
+        else:
+            with open(out_processing, "w") as outfile:
+                subprocess.Popen(" ".join(command), env=my_env, stdout=outfile, stderr=outfile).wait()  # nosec
+        return out_processing
+
+    @pyaedt_function_handler()
+    def _get_output_parameter_from_result(self, out_file, parameter_name):
+        try:
+            with open(out_file, "r") as infile:
+                lines = infile.read()
+                parmDat = lines.split("[ParmDat]:", 1)[1]
+                for keyValu in parmDat.split(","):
+                    dataAry = keyValu.split("=")
+                    if dataAry[0].strip().lower() == parameter_name.lower():
+                        return float(dataAry[1].strip().split()[0])
+            self.logger.error("Failed to compute {}. Check input parameters and retry".format(parameter_name))
+            return False
+        except IndexError:
+            self.logger.error("Failed to compute {}. Check input parameters and retry".format(parameter_name))
+            return False
+
+    @pyaedt_function_handler()
+    def compute_erl(
+        self,
+        config_file=None,
+        port_order=None,
+        specify_through_ports=None,
+        bandwidth=None,
+        tdr_duration=None,
+        z_terminations=None,
+        transition_time=None,
+        fixture_delay=None,
+        input_amplitude=None,
+        ber=None,
+        pdf_bin_size=None,
+        signal_loss_factor=None,
+        permitted_reflection=None,
+        reflections_lenght=None,
+        modulation_type=None,
+    ):
+        """Compute effective return loss (erl) using Ansys SPISIM from a S-parameter file.
+
+        Parameters
+        ----------
+        config_file : str, optional
+            Configuration file to use as a reference. The default is ``None``, in
+            which case this parameter is ignored.
+        port_order : str, optional
+            Whether to use "``EvenOdd``" or "``Incremental``" numbering for ``s4p`` files.
+            Ignored if the ports are greater than 4.
+        specify_through_ports : list, optional
+            Input and Output ports on which compute the erl. Those are ordered like ``[inp, inneg, outp, outneg]``.
+            Ignored if number of ports are 4.
+        bandwidth : float, str, optional
+            Application bandwidth: inverse of one UI (unit interval). Can be a float or str with unit ("m", "g").
+            Default is ``30e9``.
+        tdr_duration : float, optional
+            TDR duration (in second): How long the TDR tailed data should be applied. Default is ``5``.
+        z_terminations : float, optional
+            Z-Terminations: termination (Z11 and Z22) when TDR is calculated. Default is ``50``.
```

Comment:
```suggestion
            Z-terminations (Z11 and Z22) when TDR is calculated. The default is ``50``.
```
We don't want to use the colon within the description as is often done in this section.  I'm not going to comment on the following--but please revise them all as per my previous instructions. Again, the biggest issues are that the defaults    in the descriptions don't' match those shown in the method signature. Don't use a colon in a description. Don't use abbreviations without first defining them. Always use complete sentences after the first brief description.

Reply:
The default values are loaded from the cfg_dict, if the user does not define any of these parameters, it will use the default ones defined on this dictionary